.github/workflows/workflow-test-in-container.yml

```
@@ -0,0 +1,99 @@
+# Set minimum permissions to prevent unnecessary access
+permissions: {}
+
+env:
+  CI: 1
+  TURBO_TELEMETRY_DISABLED: 1
+jobs:
+  check:
+    timeout-minutes: 30
+    permissions: {}
+    runs-on: ${{ inputs.runs-on }}
+    container: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)